pkg/apiserver/registry/virtualmachinepolicy/rest.go

```diff
@@ -36,8 +36,8 @@ import (
 
 // REST implements rest.Storage for VirtualMachinePolicy.
 type REST struct {
-	vmpIndexer cache.Indexer
-	logger     logger.Logger
+	npTrackerIndexer cache.Indexer
```

Comment:
Why it is an indexer.. we should change to cache..

Reply:
The cloudResourceNpTrackerIndexer is a cache.Indexer, which is used by the rest.